src/NuGet.Core/NuGet.Commands/CommandArgs/PackArgs.cs

```diff
@@ -42,6 +42,7 @@ public class PackArgs
         public bool Tool { get; set; }
         public string Version { get; set; }
         public bool Deterministic { get; set; }
+        public IDictionary<string, string> AliasMappings { get; set; }
```

Comment:
can be done later: I know most of NuGet doesn't have good XML doc, but since this is part of our public API and therefore customers might use it, I think it would be great to have XML doc to explain public things. It might be obvious for other properties such as `Version`, `Deterministic`, and so on, but there's no way for customers to guess what this dictionary's key and value represent, so that should be documented. Consider, for example, the [xmldocs I wrote for the GetInstalledPackagesAsync model](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGetInstalledPackage.cs).